packages/core/data-transfer/src/strapi/queries/link.ts

```diff
@@ -242,7 +256,11 @@ export const createLinkQuery = (strapi: Strapi.Strapi) => {
 
         assignOrderColumns();
 
-        await connection.insert(payload).into(name);
+        if (trx) {
```

Comment:
Same as above, you can simplify the code by creating a qb variable that holds the query builder instance & run it later